src/types/connector.ts

```diff
@@ -1,11 +1,17 @@
-import { InstantSearch } from './instantsearch';
+import { Hits, InstantSearch, SearchResults } from './instantsearch';
+import { InsightsClient } from './insights';
 
-export type RenderOptions<T = unknown> = {
-  widgetParams: T;
+export type RenderOptions<WidgetParams = unknown> = {
```

Comment:
```suggestion
export type RenderOptions<TWidgetParams = unknown> = {
```

Reply:
fixed in 92690df0949a76517c46edf4d5da036bd2538a02